generate_so.py

```diff
@@ -514,4 +515,133 @@ def _make(dirpath: str) -> None:
         version='0.1',
         create_date='2019-07-21T12:15:54Z',
     )
+    _make('out/ssop')
+    _make('out/ssop/pkg')
+    generate_pkg(
+        dirpath='out/ssop/pkg',
+        author='Danilo B.',
+        # Name according to IPC7351C
+        name='SSOP{pin_count}P{pitch}_{body_length}X{lead_span}X{height}L{lead_length}X{lead_width}',
+        description='{pin_count}-pin Plastic Shrink Small Outline Package (SSOP), '
+                    'standardized by JEDEC (MO-152), variation {variation}.\\n\\n'
+                    'Pitch: {pitch:.2f} mm\\nBody length: {body_length:.2f} mm\\n'
+                    'Body width: {body_width:.2f} mm\\nLead span: {lead_span:.2f} mm\\n'
+                    'Height: {height:.2f} mm\\n'
+                    'Lead length: {lead_length:.2f} mm\\nLead width: {lead_width:.2f} mm',
+        configs=[
+            # pin count, pitch, body length, body width, total width, height
+
+            # Symbols based on JEDEC MO-152:
+            #        N    e     D    E1   E    A
+
+            # 4.40mm body width
+            #   0.65mm pitch
+            SoConfig( 8,  0.65, 3.0, 4.4, 6.4, 2.0, 'AA'),
+            SoConfig(14,  0.65, 5.0, 4.4, 6.4, 2.0, 'AB-1'),
+            SoConfig(16,  0.65, 5.0, 4.4, 6.4, 2.0, 'AB'),
+            SoConfig(20,  0.65, 6.5, 4.4, 6.4, 2.0, 'AC'),
+            SoConfig(24,  0.65, 7.8, 4.4, 6.4, 2.0, 'AD'),
+            SoConfig(28,  0.65, 9.7, 4.4, 6.4, 2.0, 'AE'),
```

Comment:
Do you have them explicitly for some reasons or would it be easier to create it with a list comprehension or similar?

Reply:
It's easier to read that way. Almost an 1:1 copy from the JEDEC standard.

Reply:
Sounds reasonable.